serde_derive_internals/Cargo.toml

```diff
@@ -15,7 +15,8 @@ rust-version = "1.56"
 path = "lib.rs"
 
 [dependencies]
-proc-macro2 = "1.0"
+# nightly version supports #![feature(proc_macro_diagnostic)]
+proc-macro2 = { version = "1.0", features = ["nightly"] }
```

Comment:
serde cannot use nightly. This is not a change we can possibly accept

Reply:
> serde cannot use nightly. This is not a change we can possibly accept

Can this warning be an  unstable only feature?  The only tools we found that can warn at compile time require Nightly Rust. The codebase already has the unstable feature flag that enables features that only compile on Nightly Rust. We've added this fix as an unstable feature

Reply:
> serde cannot use nightly. This is not a change we can possibly accept

Can this warning be an  unstable only feature?  The only tools we found that can warn at compile time require Nightly Rust. The codebase already has the unstable feature flag that enables features that only compile on Nightly Rust. We've added this fix as an unstable feature